modules/wordads/class-wordads.php

```diff
@@ -625,51 +635,120 @@ public function get_ad( $spot, $type = 'iponweb' ) {
 	 * @since 5.7
 	 */
 	public function get_ad_snippet( $section_id, $height, $width, $location = '', $css = '' ) {
+		if ( function_exists( 'is_amp_endpoint' ) && is_amp_endpoint() ) {
```

Comment:
Let's use `class_exists( 'Jetpack_AMP_Support' ) && Jetpack_AMP_Support::is_amp_request()` instead, to be consistent with other places in Jetpack.